components/style/properties/gecko.mako.rs

```diff
@@ -1338,17 +1361,7 @@ ${impl_style_struct(style_struct)}
 ${define_ffi_struct_accessor(style_struct)}
 % endfor
 
-lazy_static! {
-    pub static ref INITIAL_GECKO_VALUES: ComputedValues = ComputedValues {
-        % for style_struct in data.style_structs:
-           ${style_struct.ident}: style_structs::${style_struct.name}::initial(),
-        % endfor
-        custom_properties: None,
-        shareable: true,
-        writing_mode: WritingMode::empty(),
-        root_font_size: longhands::font_size::get_initial_value(),
-    };
-}
+static mut INITIAL_GECKO_VALUES: *mut ComputedValues = 0 as *mut ComputedValues;
 
```

Comment:
AtomicPtr scares me :) Seems fine in this situation though.

Wouldn't relaxed ordering not solve the problem of initializing it but not having the initialized value propagated? Though I guess since we call initialize before any threads (I think we do? Can't check on mobile) spin up it should be fine. Solves the overoptimization problem though.